src/compiler/literal/template/NoSubstitutionTemplateLiteral.ts

```diff
@@ -0,0 +1,13 @@
+// import * as ts from "typescript";
```

Comment:
Took me a little bit to figure out, but this is doing something like this:

```ts
let t: `test`;
```

I'll add this back.

Reply:
Oh right, sorry forgot to explain why this was commented out - I think this might not be supported by all typescript versions that ts-simple-ast supports.

Reply:
Looks like we're good. This is the 2.4 release: https://github.com/Microsoft/TypeScript/blob/8b2fe136f8cac532445bb3de89abd134592172d1/lib/typescript.d.ts#L911